Documentation/cmdref/cilium-agent.md

```diff
@@ -73,6 +73,7 @@ cilium-agent [flags]
       --encrypt-node                                  Enables encrypting traffic from non-Cilium pods and host networking
       --endpoint-interface-name-prefix string         Prefix of interface name shared by all endpoints (default "lxc+")
       --endpoint-queue-size int                       size of EventQueue per-endpoint (default 25)
+      --endpoint-status strings                       Enable additional CiliumEndpoint status features (policy,health,controllers,log,state)
```

Comment:
Why not an "all"?

Reply:
Because most users will unlikely understand the scalability consequences of enabling all options. Given the consequences, it seems more sensible to require users to opt-in by feature.